src/components/RocketCard.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const RocketCard = () => (
+  <div className="rocket-continer">
+    <div className="img-continer">
+      <img className="image" alt="rokect-img" src="https://reactjs.org/logo-og.png" />
+    </div>
+    <div className="text-continer">
```

Comment:
For better :100:  readability and to improve collaboration you can consider changing your classNames to more meaningful/descriptive ones. For example, instead of `text-continer` you can use `rocket-info` or `rocket-text` :+1:

Reply:
hi 
yeah thats a cool idea i will consider it